build.sbt

```diff
@@ -27,7 +27,7 @@ Global / licenses := Seq(
 sonatypeCredentialHost := "s01.oss.sonatype.org"
 
 ThisBuild / version := {
-  if (!sys.env.contains("CI")) "dev"
+  if (!sys.env.contains("CI")) "dev-SNAPSHOT"
```

Comment:
this was not intended per se, but I would keep it

Reply:
sure